chains/substrate/executor/executor.go

```diff
@@ -196,9 +219,11 @@ func (e *Executor) executeProposal(proposals []*chains.Proposal, signatureData *
 
 	hash, sub, err := e.bridge.ExecuteProposals(proposals, sig)
```

Comment:
We would need tracing for evm transaction sending

Reply:
But there is a tracing for this happening in this function executeProposal where this call is located at, nothing much interesting happening down there to put tracing deeper. Bcs there is basically a stack of two functions there with only one place where error can happen and only one outcome of the hash that is recorded here

Reply:
There is a transaction monitor that might resend. That module I am interested in.

Reply:
https://github.com/ChainSafe/chainbridge-core/pull/350 Added here, please review